### DISCUSSION TITLE
Can't get rid of border on barebones starter

### DISCUSSION THREAD

**Discussion by haowens** [tinacms/tinacms#4009] [Help] [answered]
When I wrap the Layout component and/or the App component in a div with a style property set to "width: 100%, height: 100% , margin: 0px and backgroundColor: black", the app has a white border around it that I can't seem to locate the cause of--does anybody know where this is coming from?

**Comment by twittkopf** [comment, marked as answer]
I guess that's the `margin` on `body` usually added by the user agent stylesheet. Drop this into your stylesheet and see if the white border goes away:

```
body {
  margin: 0;
}
```

**Reply by haowens** [reply]
thank you :)
